scale/scheduler/management/commands/scale_scheduler.py

```diff
@@ -6,6 +8,7 @@
 from django.conf import settings
 from django.core.management.base import BaseCommand
 
+from scale import __version__
```

Comment:
Prefer getting the version via the Django settings API to avoid importing higher project-level code into lower-level apps.